src/monitoring_tests/reference_solver_surplus_test.py

```diff
@@ -77,12 +78,21 @@ def compare_orders_surplus(
                     f"Absolute difference: {float(a_abs_eth):.5f}ETH ({a_abs} atoms)",
                 ]
             )
+            ref_solver_log = "\t".join(
+                [
+                    f"Tx Hash: {competition_data['transactionHash']}",
+                    f"Order UID: {uid}",
+                    "Solution providing more surplus",
+                    str(ref_solver_response[1]),
```

Comment:
For consistency with the other logging:
```suggestion
                    f"Solution providing more surplus {ref_solver_response[1]}",
```
(not sure if this works)

Reply:
Good point, and that should work actually.